### PR TITLE
added basic example for android

### LINKED ISSUE #1: Add initial code for Android
**Description**:
Add some code showing how to use in a real Android app.

**Checklist**:
- [ ] Add sub-repo
- [ ] Add README

### PR DESCRIPTION
**Pull request by ToJen**
Signed-off-by: Tomisin Jenrola <tomisin.jenrola@securekey.com>

Added simple example of using the Aries mobile bindings for Android.

Closes #1.

### CODE REVIEW

**Comment by troyronda** [comment]
@cla-bot check

**Comment by ToJen** [comment]
current screenshot can be found [here](https://github.com/ToJen/aries-examples/tree/add_initial_android_repo/android/AriesDemo#4-screenshot)

**Comment by llorllale** [comment]
@ToJen can you exclude gradle-wrapper.jar from the commit?

**Comment by ToJen** [comment]
@llorllale done

**Comment by troyronda** [comment]
I would prefer large binary files (like screenshots of the current state) to not be bundled in the same GitHub branch as the code.

https://github.com/trustbloc/aries-examples/blob/fb9ca6bc6d6d53741e85dd7efebd3f1798712f15/android/AriesDemo/assets/images/get_credentials.png

File: android/AriesDemo/app/src/main/java/com/github/trustbloc/ariesdemo/FirstFragment.java [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+    String url;
+    AriesController agent;
+
+    @RequiresApi(api = Build.VERSION_CODES.KITKAT)
+    public void setAgent() {
+        // create options
+        Options opts = new Options();
+        opts.setURL(url);
+        opts.setUseLocalAgent(false);
```
Comment (troyronda) [inline android/AriesDemo/app/src/main/java/com/github/trustbloc/ariesdemo/FirstFragment.java:42]: I would like to also see a sample usage where the local agent is enabled. This is a very important aspect to show (effectively the primary reason for the building these bindings).

**Comment by ToJen** [comment]
Here's a screenshot

Screen Shot 2020-07-16 at 10 18 50 AM

File: android/AriesDemo/app/src/main/java/com/github/trustbloc/ariesdemo/FirstFragment.java (hunk 1 above)
Comment (troyronda) [reply android/AriesDemo/app/src/main/java/com/github/trustbloc/ariesdemo/FirstFragment.java:42]: i.e., the sample should have a toggle between local agent mode and remote agent mode (with URL).

**Comment by rolsonquadras** [comment]
@ToJen The commit needs to be signed.

File: android/AriesDemo/app/src/main/java/com/github/trustbloc/ariesdemo/FirstFragment.java (hunk 1 above)
Comment (troyronda) [reply android/AriesDemo/app/src/main/java/com/github/trustbloc/ariesdemo/FirstFragment.java:42]: @ToJen will add local agent mode in next PR.

File: android/AriesDemo/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# AriesDemo Android app
```
Comment (rolsonquadras) [inline android/AriesDemo/README.md:1]: May be a link to this page in main README would be helpful for easier navigation.

File: android/AriesDemo/app/src/main/java/com/github/trustbloc/ariesdemo/FirstFragment.java (hunk 1 above)
Comment (ToJen) [reply android/AriesDemo/app/src/main/java/com/github/trustbloc/ariesdemo/FirstFragment.java:42]: confirmed

**Pull request merged**